app.js

```diff
@@ -0,0 +1,70 @@
+(function() {
+  const todos = ["Walk the dog", "Go shopping for new Jeans", "Learn web"];
+  const inputEl = document.getElementById("todo-input");
+  const addBtnEl = document.getElementById("add-btn");
+  const todoListEl = document.getElementById("todo-list");
+  let newTodo = "";
+
+  // Add functionality to HTML elements
+  inputEl.addEventListener("keyup", event => (newTodo = event.target.value));
```

Comment:
I think the double parentheses is not necessary here, it could be just 
`inputEl.addEventListener("keyup", event => newTodo = event.target.value);`
Please let me know if I am wrong.

Reply:
Prettier put it there meh